src/app/components/Footer/index.tsx

```diff
@@ -8,6 +8,8 @@ import { backend, BackendAPIs } from 'vendors/backend'
 import { MobileFooterNavigation, mobileFooterNavigationHeight } from '../MobileFooterNavigation'
 
 const githubLink = 'https://github.com/oasisprotocol/oasis-wallet-web/'
+const githubReleaseLink = (tag: string) =>
+  tag && tag !== 'v0.0' ? `${githubLink}releases/tag/${tag}` : `${githubLink}releases`
```

Comment:
I guess we don't need `tag &&` check if we expect to always have a string here. Other than that why do we care about `v0.0` value specifically? is `v0.1` valid for example?

Reply:
The motivation behind this was, that `v0.0` was set as default in `.env`. So that we don't have a broken link, I just redirected to `https://github.com/oasisprotocol/oasis-wallet-web/releases`. Right now I just set the default version to the first tag in this repository, which is kind of awkward, since this code can't be moved.